### PR TITLE
Implement update browser page

### PR DESCRIPTION
**Pull request by brniamut**
### Changes proposed in this pull request:

When a browser is not supported by jw-showcase the site will redirect to a browser update page.

### CODE REVIEW

File: views/updateBrowser/updateBrowser.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="jw-cover jw-modal ng-cloak">
+    <div class="jw-modal-container">
+        <div class="jw-modal-image">
+            <img class="jw-header-logo" ng-src="{{ ::vm.config.assets.banner }}" alt="logo"/>
+        </div>
+        <div class="jw-modal-message jw-modal-message-title">
+            Your browser is out-of-date.
```
Comment (RWOverdijk) [inline views/updateBrowser/updateBrowser.html]: Some of the features used by JW Showcase aren't support by your current browser.

File: views/updateBrowser/updateBrowser.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        </div>
+        <div class="jw-modal-message jw-modal-message-small">
+            Update your browser to view this website correctly.
```
Comment (RWOverdijk) [inline views/updateBrowser/updateBrowser.html]: Update your current browser, or install one of the other compatible browsers to fully enjoy the JW Showcase experience.

**Comment by RWOverdijk** [comment]
Some textual suggestions.

File: js/core/services/preload.service.js [hunk 3]
```diff
@@ -81,6 +81,20 @@
                     document.body.classList.remove('jw-flag-loading-config');
                 });
 
+                // if flexbox is not supported show update browser page
+                var testElement           =    document.createElement('div');
```
Comment (ChristiaanScheermeijer) [inline js/core/services/preload.service.js]: I think we should move this to a util function:

```
if (!utils.flexSupport()) {
  appStore.loading = false;

  return $state.go('updateBrowser', {directed: true});
}
```

Comment (brniamut) [reply js/core/services/preload.service.js]: Agree

File: views/updateBrowser/updateBrowser.html (hunk 1 above)
Comment (brniamut) [reply views/updateBrowser/updateBrowser.html]: agree, and done

File: views/updateBrowser/updateBrowser.html (hunk 2 above)
Comment (brniamut) [reply views/updateBrowser/updateBrowser.html]: agree and done

File: js/core/services/preload.service.js (hunk 3 above)
Comment (brniamut) [reply js/core/services/preload.service.js]: Done

**Comment by pajter** [comment]
Page not rendering in IE10.

screen shot 2017-09-25 at 14 11 20

**Comment by brniamut** [comment]
Somehow a colon was removed from the code in the commit -_-

**Comment by ChristiaanScheermeijer** [comment]
@brniamut can you make it jshint and scsslint valid?

**Comment by ChristiaanScheermeijer** [comment]
For some reason, this fails in CI but not locally.

**Pull request merged**